src/client/entity-editor/relationship-editor/relationship-editor.tsx

```diff
@@ -300,6 +300,11 @@ class RelationshipModal
 	renderEntitySelect() {
 		const {baseEntity, relationshipTypes} = this.props;
 		const {targetEntity} = this.state;
+		if (baseEntity.type === 'Series') {
+			// When the base entity is Series, remove relationship type 70 - 74.
+			// We don't want to generate entity types corresponding to these relationship type.
+			_.remove(relationshipTypes, (type) => type.id > 69 && type.id < 75);
+		}
```

Comment:
I feel this would be better in the `getValidOtherEntityTypes` helper instead, but the logic is what I'm expecting